src/controllers/routers/referees.js

```diff
@@ -58,25 +65,22 @@ const getAll = async (req, res) => {
  * @param {Request} req
  * @param {Response} res
  */
-const getReferee = async (req, res, next) => {
+const getById = async (req, res, next) => {
   const { id } = req.params;
+  if (!isValidId(id)) {
+    return next(new ResourceNotFound(`Referee ${id} does not exist`));
+  }
   try {
-    const referee = await Referee.query()
-      .select(
-        'name',
-        'email',
-        'phone',
-        'organisation',
-        'refereeId',
-        'created_at',
-        'updated_at'
-      )
-      .where('refereeId', id);
-    if (referee.length === 0) {
+    const referee = await Referee.query().findById(id);
+    if (!referee) {
       return next(new ResourceNotFound(`Referee ${id} does not exist`));
     }
     return res.status(200).json({ referee });
   } catch (err) {
+    // if input id is not an int
```

Comment:
remove this check since we're already using the isValidId function

Reply:
removed